tox.ini

```diff
@@ -53,6 +53,12 @@ use-flake8-tabs = true
 max-line-length = 100
 tab-width = 4
 
+per-file-ignores =
+    datastore/util.py:F401
+	datastore/abc.py:F401
+	datastore/typing.py:F401
+    setup.py:E121
+    other_project/*:W9
```

Comment:
There is no `setup.py` file or `other_project` directory? (Also what is warning category W9?)